content/md/en/docs/build/randomness.md

```diff
@@ -15,10 +15,10 @@ In traditional computer applications, what people call random numbers are actual
 Pseudo-randomness depends on a sufficiently random seed—provided by the user or by an external source—and an algorithm that manipulates the seed to generate a series of seemingly-random numbers. 
 The result is a pseudo-random number that is difficult to predict because of the algorithm used to generate it, but deterministic because the same seed input always produces the same series of output.
 
-However, applications that run on the blockchain are more tightly constrained because all authorities in the network must agree on any on-chain value, including any randomness data that is injected.
-Because of this constraint, you can't use real randomness directly in blockchain applications.
+However, applications that run on the blockchain are more tightly constrained. 
+Pseudo-randomness introduces a security vulnerability for situations where being able to predict the outcome of a random function can lead to gaming the system.
```

Comment:
I think the part that you removed here is important. So I'd add your new bit to it: "For this reason, Pseudo-randomness introduces a ...."

Reply:
I'm not sure I understand the statement/ logic saying: "there's a constraint, which is that nodes must agree on any on-chain value and therefore randomness can't be used on chain". It's not because of this constraint that you can't use randomness. Randomness in of itself is the constraint too, where you could argue that true randomness does not exist. 

So I'm not sure what information the lines I'm suggesting to remove are providing. Could you provide bullet points of you're interpretation?

Reply:
That makes sense. Adding an update. Thanks for your comments @Daanvdplas 🙏🏻